app/mainWindow.js

```diff
@@ -50,3 +50,7 @@ export default function showMainWindow() {
 export function getMainWindow() {
   return mainWindow;
 }
+
+export function dispatchToMainWindow(reduxAction) {
```

Comment:
want to highlight this new method to allow dispatching redux action from the main process in electron. The goal is to move wallet state management (such as backgroundMonitor.js) to the main process and optimize network activities between the wallet node and `hsd` node.